agent/command/git.go

```diff
@@ -542,6 +581,11 @@ func (c *gitFetchProject) fetchSource(ctx context.Context,
 	attempt := 0
 	return c.retryFetch(ctx, logger, true, opts, func(opts cloneOpts) error {
 		attempt++
+		if attempt > 1 {
+			opts.fallbackToFullClone = true
+			// log to splunk a warning that we are falling back to a full clone.
```

Comment:
Should there be a log here? Maybe it's not relevant given the other conversation.

Reply:
Yes that was me telling my future self to ask about how to log through to splunk-- I'm going to do the honeycomb recommendation since it would be easier and I can just query for it

Reply:
wait.. there's already a honeycomb attribute for attempt, so it wouldn't tell use that much actually.

Reply:
reminder to update this!

Reply:
The attempt attribute on the span is already added!